### PR TITLE
Gamehelp

### PR DESCRIPTION
**Pull request by roughana**
Alcazar, Cyclod, Drelbs, Gumball, Mr Cool, Pitfall II, Short Circuit, Situation Critical, Spy Hunter, Castle Wolfenstein

### CODE REVIEW

File: res/GAMEHELP/GUMBALL [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+          joystick or keyboard
+
+       button 0 open gate, 1 to aim
+
+     Production       Aim (level 4+)
+
+         E              U I O
+
+       S D F            J K L?
```
Comment (a2-4am) [inline res/GAMEHELP/GUMBALL:12]: what does the question mark mean

File: res/GAMEHELP/WOLFEN84 [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+       Q W E               I O P
+
+       A   D               K   ;
+
+       Z X C               , . /
+
+
+Space* * search body / unlock door/chest
+
+U* * * * * * * * * *use content of chest
```
Comment (a2-4am) [inline res/GAMEHELP/WOLFEN84:20]: "contents"

File: res/GAMEHELP/PITFALL.II [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+    Find gold, rat, diamond, girl, cat
```
Comment (a2-4am) [inline res/GAMEHELP/PITFALL.II:2]: "& cat"

File: res/GAMEHELP/GUMBALL (hunk 1 above)
Comment (roughana) [reply res/GAMEHELP/GUMBALL:12]: I haven't tested aim key controls myself. This was a reminder to test L to make sure it worked.

**Pull request merged**